joomla/joomla-cms
# [5][com_joomlaupdate] Deprecated: Creation of dynamic property php 8.2

Pull Request for Issue # .

### Summary of Changes

declare variables

### Testing Instructions

set error report to maximum
go to Joomla Updatde
and / or `php cli/joomla.php update:extensions:check`

### Actual result BEFORE applying this Pull Request

![image](https://github.com/joomla/joomla-cms/assets/181681/12f78ae1-9cf9-4649-a2c6-927d23855759)

![image](https://github.com/joomla/joomla-cms/assets/181681/5cfd9536-08cf-4468-b283-5954704b621b)



### Expected result AFTER applying this Pull Request

no more deprecations

### Link to documentations
Please select:
